frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -300,6 +300,28 @@ i.e.
   }
 }
 
+// Events
+.events-hero {
```

Comment:
Looks like these custom CSS items can be removed since they aren't being used. Also another reminder that we haven't documented custom css expectations https://github.com/HHS/simpler-grants-gov/issues/4097

Reply:
Ah yes, thank you !